templates/zerver/emails/email.css

```diff
@@ -6,7 +6,7 @@ img {
 }
 
```

Comment:
Add this line to the top of this file to disable the hex check rule

```scss
/* stylelint-disable color-no-hex  */
```

Also probably explain the reasoning as a comment :)